extensions/wikia/FounderEmails/FounderEmails.php

```diff
@@ -24,27 +26,22 @@
  * extension config
  */
 $wgFounderEmailsEvents = [
-	'edit'       => [
-		'className'  => 'FounderEmailsEditEvent',
-		'hookName'   => 'RecentChange_save',
-		'skipUsers'  => [ 929702 /* CreateWiki script */, 22439 /* Wikia */ ]
+	'edit' => [
+		'className' => 'FounderEmailsEditEvent',
+		'skipUsers' => [929702 /* CreateWiki script */, 22439 /* Wikia */]
```

Comment:
We now have `Fandom` and `FandomBot` users doing automated edits.

Reply:
Did you mean FANDOM and FANDOMbot? Their IDs are [32769624 and 32794352](http://wikia.com/api.php?action=query&list=users&ususers=FANDOM|FANDOMbot&format=jsonfm), if needed.